packages/jupyter-ai/jupyter_ai/personas/persona_manager.py

```diff
@@ -135,6 +139,10 @@ def _init_persona_classes(self) -> None:
                 "ERROR: Jupyter AI has no AI personas available. "
                 + "Please verify your server configuration and open a new issue on our GitHub repo if this warning persists."
             )
+
+        # TODO: check whether in the end we need a full class for this, or if we can just use a simple function that returns a list of persona classes from the local filesystem.
+        persona_classes.extend(LocalPersonaLoader(self.root_dir).load_persona_classes())
```

Comment:
We might want to wrap this in a try/except once we get a sense of what exceptions might get triggered.

Reply:
One thing we will need to think about is if a user moves the chat file while it is open (which instantiates the PersonaManager) such that a new .jupyter directory is relevant. That and we also want to reload the modules.